packages/database/schema.prisma

```diff
@@ -352,30 +351,12 @@ model ActionClass {
   environment    Environment     @relation(fields: [environmentId], references: [id], onDelete: Cascade)
   environmentId  String
   surveyTriggers SurveyTrigger[]
-  actions        Action[]
 
   @@unique([key, environmentId])
   @@unique([name, environmentId])
   @@index([environmentId, createdAt])
 }
 
-model Action {
-  id            String      @id @default(cuid())
-  createdAt     DateTime    @default(now()) @map(name: "created_at")
-  actionClass   ActionClass @relation(fields: [actionClassId], references: [id], onDelete: Cascade)
-  actionClassId String
-  person        Person      @relation(fields: [personId], references: [id], onDelete: Cascade)
-  personId      String
-  /// @zod.custom(imports.ZActionProperties)
-  /// @zod.custom(imports.ZActionProperties)
-  /// [ActionProperties]
```

Comment:
These types should also be removed from `packages/database/json-types.ts` & `packages/database/zod-utils.ts`